packages/kbn-test/src/functional_tests/tasks.js

```diff
@@ -0,0 +1,114 @@
+import { relative, resolve } from 'path';
+import Rx from 'rxjs/Rx';
+import getopts from 'getopts';
+import { withProcRunner } from '@kbn/dev-utils';
+
+import {
+  withTmpDir,
+  runKibanaServer,
+  runElasticsearch,
+  runFtr,
+  log,
+  KIBANA_ROOT,
+  KIBANA_FTR_SCRIPT,
+  FUNCTIONAL_CONFIG_PATH,
+  API_CONFIG_PATH,
+} from './lib';
+
+import { readConfigFile } from '../../../../src/functional_test_runner/lib';
+
+const SUCCESS_MESSAGE = `
+
+Elasticsearch and Kibana are ready for functional testing. Start the functional tests
+in another terminal session by running this command from this directory:
+
+    node ${relative(process.cwd(), KIBANA_FTR_SCRIPT)}
+
+`;
+
+// Takes in multiple configs, comma separated
+// Runs servers and tests for each config
+export async function runTests(
+  configPaths = [FUNCTIONAL_CONFIG_PATH, API_CONFIG_PATH],
+  runEs = runElasticsearch,
+  runKbn = runKibanaServer
+) {
+  const configs = [];
+
+  // Make a list of config paths based on --config or defaults
+  try {
```

Comment:
Take another look? I made some cli functions using only getopts. Commander would be overkill at the moment because we also need special args to be passed to functional_test_runner. I think this should be resolved in a followup.